api/rest/property_value.md

```diff
@@ -6,156 +6,568 @@ parent: REST
+When actually putting together your request, you can choose from two options. If you intend to use only a small number of parameters, you may want to use the simpler formatting offered by the GET method, which makes requests against the main endpoint while altering the query parameters incorporated into the URL. For more complex queries, or if you prefer to utilize a static URL, a POST request likely makes more sense. To use it, make a POST request against the main endpoint while changing the fields of the JSON body it sends.
 
-## Success Response
+## What to expect in the response
 
-### **Code**: `200 OK`
-
-**Response content example**
+Your response will always look like this:
 
 ```json
 {
     "payload": "<payload string>",
 }
 ```
 
-The equivalent JSON of "payload string" is as follows.
+Here `"<payload string>"` is a long encoded JSON string, whose structure changes depending on whether the response contains node references. You can run `JSON.parse()` on the `payload` field to retrieve the data. For example, in JavaScript: `var data = JSON.parse(response['payload'])`.
+
+After decoding the response payload string, there are two possible structures it could adhere to.
 
-**Example 1:** property values that are not nodes.
+**Structure 1:** Decoded response payload string for property values that are not node references.
 
 ```json
 {
-    "geoId/05": {
-        "out": [
-          {
-            "value": "Arkansas",
-            "provenanceId": "dc/sm3m2w3"
-          }
-        ]
-    },
-    "geoId/06": {
-        "out": [
+    "<dcid>": {
+        "direction": [
           {
-            "value": "California",
-            "provenanceId": "dc/sm3m2w3"
-          }
+            "value": "string",
+            "provenanceId": "string"
+          },
+          ...
         ]
     },
+    ...
 }
 ```
 
-**Example 2:** property values that are node references.
+**Structure 2:** Decoded response payload string for property values that are node references.
 
 ```json
 {
-    "geoId/06":
+    "<dcid>":
         {
-        "in": [
+        "direction": [
             {
-              "dcid": "ipedsId/486619",
-              "name": "Academy of Professional Cosmetology",
-              "provenanceId": "dc/89fk9x3",
-              "types": ["CollegeOrUniversity"]
+              "dcid": "string",
+              "name": "string",
+              "provenanceId": "string",
+              "types": "string[]"
             },
-            {
-              "dcid": "ipedsId/485731",
-              "name": "The Beauty School",
-              "provenanceId": "dc/89fk9x3",
-              "types": ["CollegeOrUniversity"]
-            }
+            ...
         ]
+    },
+    ...
+}
+```
+
+**NOTE:** The `provenanceId` is the DCID of the provenance for the corresponding value.
+
+## Example requests and responses
+
+### Example 1: Retrieve the two common names of the country of Côte d'Ivoire.
```

Comment:
Retrieve the common name(s) ? I don't want users to think we are doing limit=2

Reply:
I will remove "two" from this header.